demos/src/Nodes/Mention/Vue/suggestion.js

```diff
@@ -63,8 +78,8 @@ export default {
       },
 
       onExit() {
-        popup[0].destroy()
         component.destroy()
+        component.element.remove()
```

Comment:
I find this a bit strange, why did we not have to do this before? Shouldn't destroying the component also unmount the element? Should we fix this somehow?

Reply:
Nice find - yeah we don't need that one.